.circleci/config.yml

```diff
@@ -28,3 +28,6 @@ jobs:
     - persist_to_workspace:
         root: ~/go/src/github.com/Clever
         paths: "."
+    - run: if [ "${CIRCLE_BRANCH}" == "master" ]; then cat ./swagger.yml | grep "^  version:" | cut -d":" -f2 | tr -d " " > ./VERSION; fi;
```

Comment:
This won't work because there is no swagger.yml in this repo. The VERSION file already exists so you just need the line below